projectRM/projectRM/ViewController.swift

```diff
@@ -6,6 +6,7 @@
 //
 
 import UIKit
+
 /// Экран, который отвечает за ...
```

Comment:
```suggestion
/// Экран, который отвечает за ...
```
за что отвечает этот экран?